workflow.py

```
@@ -0,0 +1,325 @@
+    "backend-dev": "개발자",
+    "reviewer": "코드 리뷰어",
+    "doc-slave": "문서화 노예",
+    "doc-reviewer": "문서 리뷰어",
+}
+
+
+def load_persona(step: str) -> str:
+    """페르소나 파일을 읽어 시스템 프롬프트로 반환한다."""
+    persona_file = PERSONAS[step]
+    if not persona_file.exists():
+        raise FileNotFoundError(f"페르소나 파일을 찾을 수 없습니다: {persona_file}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)